index.html

```diff
@@ -1052,6 +1097,160 @@ <h4 id="Abilities">Abilities</h4>
+         <p>
+           You can unlock a legendary ability at level 1, and an additional one every 5 levels.  
+           These give a significant ability or passive to your character.
+           Some can be taken multiple times.
+         </p>
+         <table class="center">
+            <tr>
+               <td><strong>Name</strong>
+               </td>
+               <td><strong>Desc</strong>
+               </td>
+               <td><strong>Requirements</strong>
+               </td>
+            </tr>
+            <tr>
+               <td>Legendary Endurance
+               </td>
+               <td>
+                  Increase your max HP by additional CON value. This additional HP is recalculated as normal if 
+                  your CON value changes.
+               </td>
+               <td>Can be taken up to 3 times. You may only take either Legendary Endurance or Legendary Power
+                  abilities a total of 3 times. Such as taking Legendary Endurance 3 times and not taking Legendary Power (or vice versa),
+                  or 2 Legendary Endurance and 1 Legendary Power (or vice versa).
+               </td>
+            </tr>
+            <tr>
+               <td>Legendary Power
+               </td>
+               <td>
+                  Light / Medium / Heavy values increase by 1 / 2 / 3 respectively.
+               </td>
+               <td>Can be taken up to 3 times. You may only take either Legendary Endurance or Legendary Power
+                  abilities a total of 3 times. Such as taking Legendary Endurance 3 times and not taking Legendary Power (or vice versa),
+                  or 2 Legendary Endurance and 1 Legendary Power (or vice versa).
+               </td>
+            </tr>
+            <tr>
+               <td>
+                  Legendary Empowered Resistance Drain
+               </td>
+               <td>
+                  <p>
+                  Upgrade to Empowered Resistance Drain. When you use Empowered Resistance Drain, the number
+                  of resistance points spent grants another creature of your choice bonus damage on the next instance of damage they 
+                  inflict before the start of your next turn. 1, 2 or 3 resistance points spent results in additional Light, Medium, or Heavy damage
+                  respectively - the type of damage is the same as what the attack deals.
+                  </p>
+                  Any creature can only give this effect once per turn, and any creature can only have one instance of this effect on
+                  them at any given time. 
+               </td>
+               <td>Can be taken once.
+               </td>
+            </tr>
+            <tr>
+               <td>
+                  Legendary Empowered Ability
+               </td>
+               <td>
+                  <p>
+                  Upgrade to Empowered Ability. When you use Empowered Ability, the number
```

Comment:
So this grants max an additional 3 damage?  Which you only get when you do empowered ability?  Just getting the Power upgrade gives you that in one attack if you use Heavy.  

The advantage can be kinda cool - still don't see it being that useful for the most part but I would remove the damage options.  It adds complexity and is laughably bad.     Not sure this is the one either

Reply:
With the addition of the glancing damage as well the need for advantage also is pretty minial imo as a reason to empower